fwd

```diff
@@ -1,15 +1,23 @@
 #!/usr/bin/env bash
 
+# Ports - used on docker-compose.yml
 export HTTP_PORT=${HTTP_PORT:-80}
 export MYSQL_PORT=${MYSQL_PORT:-3306}
+# Host user for mapping execution within the containers
 export ASUSER=${ASUSER:-$UID}
+# Docker Compose flags
 export COMPOSE_EXEC_FLAGS=${COMPOSE_EXEC_FLAGS:-""}
+# SSH key for mapping within container so compose/npmm can have access to secured sources
 export SSH_KEY_PATH=${SSH_KEY_PATH:-"$HOME/.ssh/id_rsa"}
 
 if [ -f .env ]; then
     source .env
 fi
 
+# Images to use out of docker-compose.yml; can be overriden on .env
+FWD_IMAGE_NODE=${$FWD_IMAGE_NODE:-"fireworkweb/node:alpine"}
```

Comment:
only thing is maybe we could do this before `source .env`, so we ca add these variables to our .env too